www/running.inc

```diff
@@ -11,6 +11,8 @@
             $noanalytics = true;
             $page_keywords = array('WebPageTest','Website Speed Test','Page Speed');
             $page_description = "Running website performance test.";
+	    
+	    require_once WWW_PATH . '/templates/layouts/google-tag-manager.inc';
```

Comment:
@mjkozicki  this is already included in head.inc but changing $noanalytics = false; will import also other scripts, so probably better to just include gtm here?